crate/operator/backup.py

```diff
@@ -240,22 +241,24 @@ def create_backups(
                     image_pull_secrets,
                     has_ssl,
                 ),
-            ),
-            call_kubeapi(
-                apps.create_namespaced_deployment,
-                logger,
-                continue_on_conflict=True,
-                namespace=namespace,
-                body=get_backup_metrics_exporter(
-                    owner_references,
-                    name,
-                    labels,
-                    http_port,
-                    prometheus_port,
-                    backup_aws,
-                    image_pull_secrets,
-                    has_ssl,
+            )
+            deployment = (
```

Comment:
With this type of formatting it is not obvious that `deployment` will end up as type `tuple` - note the comma after the `call_kubeapi` call.

```
deployment = (
    await call_kubeapi(
        ...
    ),
)
```

I suggest to format like this:

```
deployment = await call_kubeapi(
...
)
```

I am wondering why mypy does not catch this case? :thinking:

Reply:
Wow. Nice catch!